src/AnnotationThread.js

```diff
@@ -267,21 +278,19 @@ class AnnotationThread extends EventEmitter {
             return Promise.reject();
         }
 
-        // Delete annotation on client
-        this.annotations = this.annotations.filter(({ id }) => id !== annotationIDToRemove);
+        // Delete matching comment from annotation
+        this.comments = this.comments.filter(({ id }) => id !== annotationIDToRemove);
```

Comment:
Would it make sense to make the terminology a bit more consistent in these variables and method names? i.e. this method is `deleteAnnotation` but it looks like it's removing a comment from the comments list (mainly). Similar idea for `saveAnnotation` which feels more like saving a comment

Reply:
I'll rename all the Threads to Annotations in a separate PR as well to clear up that confusion